src/modules/formlyBootstrap.js

```diff
@@ -1,20 +1,19 @@
 angular.module('formlyBootstrap', ['formly'], function configFormlyVanilla(formlyConfigProvider) {
   'use strict';
 
-  formlyConfigProvider.setWrapper([
-    {
+  formlyConfigProvider.setWrapper([{
       name: 'bootstrapLabel',
       templateUrl: 'wrappers/formly-wrappers-bootstrap-label.html'
-    },
+        },
     {
       name: 'bootstrapHasError',
       templateUrl: 'wrappers/formly-wrappers-bootstrap-has-error.html'
-    }
-  ]);
+        }
+    ]);
```

Comment:
Sorry to be so nit-picky, but the whitespace around here is all messed up. Please make sure that you're indenting with 2 spaces. Thanks!